modules/core/src/lib/fetch/fetch-file.js

```diff
@@ -14,5 +15,9 @@ export async function fetchFile(url, options) {
   }
   url = resolvePath(url);
   // TODO - SUPPORT reading from `File` objects
-  return fetch(url, options);
+  const response = await fetch(url, options);
+  if (!response.ok && options.throws) {
+    throw new Error(await getErrorMessageFromResponse(response));
```

Comment:
Seeing as `fetchFile` already offers a bulleted lists of "improvements" compared to vanilla `fetch`, adding improved error handling to the list is fine. 

However I think we should briefly document this error handling behavior in `fetchFile` docs, as this behavior diverges from `fetch` semantics (we no longer leave the response.ok check to the app).